Documentation/ContentBlocks/Index.rst

```diff
@@ -0,0 +1,153 @@
+:navigation-title: Custom Content Blocks
+..  include:: /Includes.rst.txt
+
+..  _content-blocks:
+
+=====================
+Custom Content Blocks
+=====================
+
+When we filled the `stage with content <https://docs.typo3.org/permalink/t3sitepackage:slide-mode>`_
+we noticed that the content does not look like the content within the original
+HTMl layout.
+
+This is because the stage does not contain normal content, as provided by
+Fluid-Styled Content, but a Jumbotron element or a slider.
+
+..  note::
+    The extension :composer:`friendsoftypo3/content-blocks` is not an official
+    part of the TYPO3 Core yet. It is also possible to
+    `Create a custom content element with TYPO3 Core only <https://docs.typo3.org/permalink/t3coreapi:adding-your-own-content-elements>`_.
+
+..  _content-blocks-installation:
+
+Install extension Content Blocks
+================================
+
+The extension :composer:`friendsoftypo3/content-blocks` is an extension that
+is not part of the TYPO3 Core but maintained by a group of community members.
+
+There are plans to integrate this extension into the Core, however at the
+time of writing there are no finite decisions yet.
+
+First install the extension :composer:`friendsoftypo3/content-blocks`:
+
+..  code-block:: bash
+
+    ddev composer req friendsoftypo3/content-blocks
+
+Set up the extension and delete all caches:
+
+..  code-block:: bash
+
+    ddev typo3 extension:setup
+    ddev typo3 cache:flush
+
+..  _content-blocks-jumbotron:
+
+The jumbotron content block
+===========================
+
+The site package you generated in step
+`Generate a site package <https://docs.typo3.org/permalink/t3sitepackage:minimal-design>`_
+comes with two content elements. We look at the more basic content elements first.
+
+You can now replace the content element in the area "Stage" of start page with
+one of type "Jumbotron".
+
+You can now create a jumbotron with a button and a link.
+
+Directory :path:`packages/my_site_package/ContentBlocks/ContentElements` contains
+one directory for each Content Block that can be used as normal Content Elements.
+
+..  _content-blocks-jumbotron-directory:
+
+Directory structure of a content block
```

Comment:
I'd like to write Content Block consistently in upper camel case, as it is a special name.